extensions/dacpac/src/utils.ts

```diff
@@ -22,14 +23,25 @@ export function getPackageInfo(packageJson: any): IPackageInfo | undefined {
 }
 
 /**
- * Map an error message into a short name for the type of error.
- * @param msg The error message to map
+ * Get file size from the file stats using the file path uri
+ * @param uri The file path
  */
-export function getTelemetryErrorType(msg: string): string {
-	if (msg && msg.indexOf('Object reference not set to an instance of an object') !== -1) {
-		return 'ObjectReferenceNotSet';
+export async function getFileSize(uri: string): Promise<string | undefined> {
```

Comment:
This function should either never throw an exception (return some default value if an error occurs) or the telemetry events which call it should handle exceptions appropriately. Telemetry events should never result in exceptions being thrown as that can mess up functionality of the components.